app/policies/note_policy.rb

```diff
@@ -0,0 +1,21 @@
+class NotePolicy < ApplicationPolicy
+  def index?
+    true
+  end
+
+  def show?
+    true
+  end
+
+  def create?
+    signed_in?
+  end
+
+  def update?
+    signed_in? && user == record.author
```

Comment:
```suggestion
    user == record.author
```
We can just check that it's a user's note since we use `authenticate_user!` method for this action in the controller.